### PR TITLE
[s] real final april fools day shit

### PR DESCRIPTION
**Pull request by Bawhoppen**
mmmmmmmmmmmmmmm

### CODE REVIEW

**Comment by TheNightingale** [comment]
Give them to the HoS, Warden and Officers too, then. Second amendment says so.

**Comment by Bawhoppen** [comment]
theres already elite sec

**Comment by Bawhoppen** [comment]
ok i fixed my mistake earlier whatevs

File: code/modules/jobs/job_types/security.dm [hunk 1]
```diff
@@ -334,7 +334,7 @@ var/list/sec_departments = list("engineering", "supply", "medical", "science")
 	spawn_positions = 5
 	supervisors = "the head of security"
 	selection_color = "#ffeeee"
-	minimal_player_age = 1000000000
+	minimal_player_age = 1
```
Comment (PKPenguin321) [inline code/modules/jobs/job_types/security.dm]: muther fuker

**Pull request merged**